resources/assert_lib.brs

```diff
@@ -84,6 +84,15 @@ function __formatError(error)
     numStackFrames = 3
     stackFrames = _brs_.getStackTrace(numStackFrames, fileFilters)
 
+    ' Convert to strings so we can see diffs
+    if GetInterface(error.expected, "ifString") = invalid then
+        error.expected = formatJson(error.expected)
```

Comment:
Neat use of `formatJson`!  I assume this is because YAML is a strict superset of JSON, so it winds up still being valid YAML?

Reply:
Great question, I'm assuming that's what's happening as well!